great-idea/css/index.css

```diff
@@ -63,4 +63,234 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.navigation-bar {
+	display: flex;
+	padding: 10px;
+	justify-content: center;
+	/*width: 850px;
+	top: 0;
+	border-bottom: 0px solid black;
+	position: fixed;
+	background-color: white;
+	width: 900px;
+	z-index: 5;*/
+}
+
+
+body {
+	margin: 0 auto;
+	width: 880px;
+	font-size: 16px;
+}
+
+header {
+	background-color: white;
+	width: 100%;
+	align-items: center;
+	height: 36px;
+}
+
+header nav a {
```

Comment:
I love the use of specificity on this one, really good and should keep any issues in the future from happening if you have similar tags, I would however suggest to instead go with classes.. this way you can still user header nav a because you'll be calling them with unique classes instead.